Assets/UnZENity-Core/Scripts/Manager/GUZBootstrapper.cs

```diff
@@ -116,6 +117,11 @@ private static void LoadDialogs()
             infoInstances.ForEach(i => GameData.Dialogs.Instances.Add(i));
         }
         
+        private static void LoadSubtitles()
+        {
+            GameData.Dialogs.CutsceneLibrary = new($"{GameGlobals.Settings.Gothic1Path}\\_work\\DATA\\scripts\\Content\\CUTSCENE\\Ou.csl");
```

Comment:
Forgot about the adapter path

```suggestion
            GameData.Dialogs.CutsceneLibrary = new($"{GameContext.GameVersionAdapter.RootPath}\\_work\\DATA\\scripts\\Content\\CUTSCENE\\Ou.csl");
```

Reply:
Great catch! Thanks